rubin_sim/maf/metrics/visit_groups_metric.py

```diff
@@ -223,8 +238,8 @@ def _in_window(self, visits, nights, night, window, min_n_visits):
         return visits[condition][condition2], nights[condition][condition2]
 
     def reduce_n_visits_in_window(self, metricval):
-        """Reduce to max number of total visits on all nights with more than minNVisits,
-        within any 'window' (default=30 nights)."""
+        """Reduce to max number of total visits on all nights with more than
+        minNVisits, within any 'window' (default=30 nights)."""
```

Comment:
The """ should go on a new line. (rubin docstyle )

Reply:
would it be okay to leave this update for the issue focused on aligning with with developer guide?

Reply:
ok.